### PR TITLE
[Mail] New mark all as read command 

### PR DESCRIPTION
**Pull request by tleo19**
## Description

Add a new `Mark all as read` Command which Mark as read every message from every mailbox from every account 

## Screencast

https://github.com/raycast/extensions/assets/65966422/2633900b-77b3-4156-b016-83c13d58f507

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Looks good to me @tleo19 🔥 

 @yug2005 do you mind checking this?

**Review by ikupenov — APPROVED** [review]

File: extensions/mail/src/mark-all-as-read.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      set visible of window 1 to false 
+      set read status of every message of every mailbox of every account where its read status = false to true
+      set visible of window 1 to true
+    end tell
+`;
+
+  runAppleScript(script);
+}
+
+export default async function command() {
```
Comment (ikupenov) [inline extensions/mail/src/mark-all-as-read.tsx:16]: In order to keep it consistent, can you rename this to `MarkAllAsRead`?

**Review by ikupenov — CHANGES_REQUESTED** [review]

**Comment by pernielsentikaer** [comment]
There is a few comments left @tleo19, could you check them out 🙂

File: extensions/mail/src/mark-all-as-read.tsx (hunk 1 above)
Comment (tleo19) [reply extensions/mail/src/mark-all-as-read.tsx]: Hey @ikupenov and @pernielsentikaer, thanks for the feedback. 
I just pushed a new commit with the changes.

**Review by ikupenov — APPROVED** [review]
LGTM 👍

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**